style.css

```diff
@@ -0,0 +1,352 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 62%;
+  background-color: #e5e5e5;
+}
+
+.menu-wrap {
+  position: absolute;
+  width: 375px;
+  height: 70px;
+  padding: 20px;
+  background-color: white;
+}
+
+.brand {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.bar {
+  width: 30px;
+  height: 3px;
+  margin: 5px;
+  display: block;
+  background-color: #6070ff;
+}
+
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.wrapper {
+  height: 718px;
+  background-image: url(/images/header.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: white;
+  border-bottom-left-radius: 8em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px;
+}
+
+.title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.intro {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.connect {
+  width: 117px;
+  height: 24px;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.socials ul {
+  width: 50%;
+  display: flex;
+  justify-content: space-between;
+  padding: 2px;
+  align-items: center;
+  flex-direction: row;
+}
+
+li {
+  list-style: none;
+}
+
+.cards {
+  display: flex;
+  gap: 88px;
+  flex-direction: column;
+  padding-top: 40px;
+}
```

Comment:
- [ ] Kindly use the `display: grid;` instead of  `display: flex;` because it's in the requirements

![grid](https://user-images.githubusercontent.com/95412943/219850025-28822ad5-78d3-4cde-a5b9-bb142044c961.PNG)